go-controller/pkg/config/config.go

```diff
@@ -2023,6 +2023,11 @@ func buildOvnAuth(exec kexec.Interface, northbound bool, cliAuth, confAuth *OvnA
 			return nil, fmt.Errorf("certificate or key given; perhaps you mean to use the 'ssl' scheme?")
 		}
 		auth.Scheme = OvnDBSchemeUnix
+		if northbound {
+			auth.Address = "unix:/var/run/ovn/ovnnb_db.sock"
```

Comment:
@zshi-redhat you could use `direction` here to fmt.Sprintf() the socket address.

Reply:
updated.